packages/xo-web/src/xo-app/pool/tab-advanced.js

```diff
@@ -192,6 +193,23 @@ class PoolMaster extends Component {
   }
 }
 
+@connectStore(() => ({
+  defaultSr: createGetObject((_, { pool }) => pool.default_SR),
+}))
+class SelectDefaultSr extends Component {
+  _onChange = sr => setDefaultSr(sr)
+
+  render() {
+    const { pool, defaultSr } = this.props
+
+    return (
+      <XoSelect onChange={this._onChange} value={pool.default_SR} xoType='SR'>
+        {defaultSr.name_label}
+      </XoSelect>
```

Comment:
There is a reason of using `pool.default_SR` instead of `defaultSr.id` for the `value` prop?